lib/cylc/cfgspec/gcylc.py

```diff
@@ -41,16 +42,20 @@
 USER_FILE = os.path.join(os.environ['HOME'], '.cylc', 'gcylc.rc')
 
 SPEC = {
-    'initial views': vdr(vtype='string_list', default=["text"]),
-    'ungrouped views': vdr(vtype='string_list', default=[]),
-    'use theme': vdr(vtype='string', default="default"),
     'dot icon size': vdr(
         vtype='string',
         default="medium",
         options=["small", "medium", "large", "extra large"]),
+    'initial side-by-side views': vdr(vtype='boolean', default=False),
+    'initial views': vdr(vtype='string_list', default=["text"]),
     'sort by definition order': vdr(vtype='boolean', default=True),
+    'sort column': vdr(
+        vtype='string',
+        default='none',
+        options=[heading for heading in ControlTree.headings if heading is not
+                 None] + ['none']),
```

Comment:
Probably clearer to use a different name for the temporary variable here?  `heading for h in ... if h is not...`